src/shared/battery-util.c

```diff
@@ -267,13 +267,11 @@ int battery_is_discharging_and_low(void) {
         }
 
         /* If we found a battery whose state we couldn't read, don't assume we are in low battery state */
-        if (unsure)
+        if (unsure) {
+                log_debug("Found battery with unreadable state, assuming not in low battery state.");
```

Comment:
log_device_debug(dev, …)

Reply:
Hmm, what if there are multiple batteries with unreadable state? We wait until the loop finishes after all 🤔

Reply:
And I don't think we should do the logging in the loop, because we might still find charged batteries later.